code/modules/mob/mob.dm

```diff
@@ -493,9 +493,14 @@
 
 	//SKYRAT EDIT ADDITION
 	if(result.len)
-		for(var/i = 1, i <= result.len, i++)
-			if(!findtext(result[i], "<hr>"))
+		for(var/i = 1, i <= length(result), i++)
```

Comment:
use `for(var/i in startnum to endnum)` formatting, in this case it would be `for(var/i in 1 to length(result))`

Reply:
I actually wasn't sure if this would work with the `i--` rewinding but amazingly it does.